### PR TITLE
reorg docs

### PR DESCRIPTION
**Pull request by Pana**
This change is [Reviewable](https://reviewable.io/reviews/conflux-chain/js-conflux-sdk/97)

### CODE REVIEW

File: docs/interact_with_contract.md [hunk 1]
```diff
@@ -1,6 +1,18 @@
 # Interact with contract
 In Conflux world you may often need to interact with contracts, with JS SDK this can be done very easy.
 
+
+>Note: when interacting with contract and if your parameter is bigger than `Number.MAX_SAFE_INTEGER`, you should use string represention of the number or BigInt.
+
```
Comment (GeekBerry) [inline docs/interact_with_contract.md:6]: 最好给出完整可执行的例子

File: docs/interact_with_contract.md [hunk 2]
```diff
@@ -1,6 +1,18 @@
...
+```js
+// use string
+await contract.deposit('90071992547409910');
```
Comment (GeekBerry) [inline docs/interact_with_contract.md:9]: 查询例子不要使用 write 函数做例子

File: docs/interact_with_contract.md (hunk 1 above)
Comment (GeekBerry) [reply docs/interact_with_contract.md:6]: const staking = conflux.InternalContract('Staking');

  const balance = await staking.getStakingBalance('0x1dcc30684fd998b2bb71613f68b26fccb6f9a591');

File: docs/overview.md [hunk 3]
```diff
@@ -31,7 +28,17 @@ const conflux = new Conflux({
 Besides `url` and `logger` you can pass [other options](../api.md) to initialize a Conflux object
 
 
-### Send JSON-RPC request
+## Add account
+Private keys are required to approve any transaction made on your behalf, `conflux.wallet` provide utility help you manage your accounts
+
+```js
+conflux.wallet.addPrivateKey('your-private-key');
```
Comment (GeekBerry) [inline docs/overview.md:35]: 给个 Hex64 做例子, 如 '0x0123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef'

File: docs/overview.md [hunk 4]
```diff
@@ -89,6 +101,24 @@ console.log(max.toString()); // directly log a jsbi is very ugly
 // JSBI(2) [ -1, 2097151, sign: false ]
 ```
 
+Note: When a js integer is bigger than `Number.MAX_SAFE_INTEGER` (2^53 - 1 or 9,007,199,254,740,991) you should use [`BigInt`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt) to contain it.
```
Comment (GeekBerry) [inline docs/overview.md:104]: 没必要在 sdk 文档介绍 BigInt 吧?

File: .gitignore [hunk 5]
```diff
@@ -3,3 +3,4 @@
 /coverage/
 /esm/
 /lib/
+/docs/tmp.md
```
[the remark this answers is not in the document]
Comment (Pana) [reply .gitignore]: done

File: docs/interact_with_contract.md (hunk 2 above)
Comment (Pana) [reply docs/interact_with_contract.md:9]: This is write example

File: docs/overview.md (hunk 3 above)
Comment (Pana) [reply docs/overview.md]: Done

File: docs/overview.md (hunk 4 above)
Comment (Pana) [reply docs/overview.md:104]: A lot SDK user make mistake using number for very very big number

File: docs/interact_with_contract.md (hunk 1 above)
Comment (Pana) [reply docs/interact_with_contract.md:6]: Here the example is used to explain how to pass big number parameter

**Pull request merged**